adhocteam/Head-Start-TTADP
# [TTAHUB-220] Regional Dashboard: Add accessibility option for Topics Graph

## Description of change
Add a table view for the topics bar graph.

## How to test
Visit the dashboard. Check out the bar graph and click on the "View Accessible Data" link. Make sure the table data accurately reflects the data on the chart.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-220

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
